components/base/Footer.vue

```diff
@@ -167,12 +167,11 @@
 </template>
 
 <script>
-import {
-  DFooter,
-  CCVLogo,
-  BrownLogo,
-  DIcon
-} from '@brown-ccv/disco-vue-components';
+import DFooter from '@/components/disco-vue-components/d-footer';
```

Comment:
maybe just incorporate the `DFooter` directly in here instead of as an import (same as navbar)